CleanSlate/common/buildings/00_castleCulture.txt

```diff
@@ -42,8 +42,7 @@ castle = {
 			OR = {
 				culture = english
 				culture = welsh
-				has_building = ca_culture_saxon_english_1
-				has_building = ca_culture_saxon_english_2
+				FROMFROM = { has_building = ca_culture_saxon_english_2 }
```

Comment:
As I understand it, this will prevent the building from being able to be constructed, in general, because they no longer meet the potential before the building is constructed.